pkg/occlient/occlient.go

```diff
@@ -1507,6 +1507,15 @@ func (c *Client) WaitAndGetPod(selector string) (*corev1.Pod, error) {
 	}
 	defer w.Stop()
 
+	// Retrieve the PodTimeout configuration parameter
+	podTimeout := time.Duration(config.DefaultPodTimeout * time.Second)
```

Comment:
Yeah, I think that this could be refactored a ton, but at the moment, I thought of implementing this the same way that the current `Timeout` is. 

Ideally, having this separated into a separate function would be better.

I'll work on getting this refactored :+1: